Scripts/script-Base64ListToFile.yml

```diff
@@ -13,12 +15,17 @@ script: |-
 
   # get the list
   res = demisto.executeCommand('getList', {'listName': demisto.args()['listname']})
+  if res['Type'] == entryTypes['error']:
+    return_error("error reading list %s from demisto" % demisto.args()['listname'])
+    sys.exit(0)
 
   # convert base64 file to binary file
   bin_file = base64.decodestring(res[0]['Contents'])
+  if demisto.args()['isZipFile'] == 'yes':
```

Comment:
This might fail if the argument is an empty string (I know that there is default value). I think it's better to use `demisto.args().get('isZipFile', 'no')`

Reply:
can't failed - there is a default